readme.md

```diff
@@ -660,6 +661,18 @@ import {
 } from '@briza/illogical'
 ```
 
+#### Strict Type Comparison
+
+Configures strict data type check in comparison expressions
+
+```typescript
+strictTypeComparison: boolean;
+```
+
+**Default strict type comparison:**
```

Comment:
Mention here what is the actual default value, to be clear. It is set to TRUE by default.

Reply:
d448793